scripts/qa_crowbarsetup.sh

```diff
@@ -2461,6 +2461,15 @@ function enable_ssl_generic()
             fi
             $p "$a['api']['protocol']" "'https'"
         ;;
+        ceph)
+            if ! iscloudver 7plus ; then
+                return
```

Comment:
@bmwiedemann You are right, it's also there for cloud6. Where do I find cloud5 though? I mean, what branch in crowbar-ceph is it in?